docs/versioned_docs/version-2.11.0/additional-info/kb.md

```diff
@@ -1,6 +1,11 @@
 ---
 id: kb
 title: Knowledge Base
+keywords: 
+ - Knowledge base
+ - OpenEBS Knowledge base
+description: The knowledge base may include Summary, manuals, troubleshooting guides, runbooks and other information that users may want or need to know.
```

Comment:
```suggestion
description: The knowledge base may include summaries, manuals, troubleshooting guides, runbooks, and other information that users may want or need to know.
```

Reply:
Since this a SEO description of the page, I think it's better to skip the word `may`.  This should say the definite context of the page.